app/helpers/matches_helper.rb

```diff
@@ -0,0 +1,69 @@
+module MatchesHelper
+
+  def init_matches_data(matches)
+    arr = []
+    matches.each do |match|
+      match.results['questions_answered'].each do |data|
+        arr.push(data)
+      end
+    end
+    return arr
+  end
+
+  def select_questions(id, matches)
+    questions = Question.where(:quiz_id => id)
+    matches_questions = init_matches_data(matches)
+    questions_list = []
```

Comment:
É uma boa prática pular linha antes de começar um novo bloco de código, ajuda na visualização
Mas é só um detalhe ;)
```suggestion
    questions_list = []
    
```

Reply:
Dá uma olhada nos outros lugares que seria bom pular linha